packs/data/other-effects.db/effect-unconscious-until-end-of-next-turn.json

```diff
@@ -0,0 +1,44 @@
+{
+    "_id": "tchHfkOrzosRnPzz",
+    "img": "systems/pf2e/icons/effects/crtical-hit-fumble.webp",
+    "name": "Effect: Unconscious until end of next turn",
+    "system": {
+        "badge": null,
+        "description": {
+            "value": "<p><strong>Effect applied by Critical Hit/Fumble Deck</strong><br />Unconscious until end of next turn</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "key": "GrantItem",
+                "uuid": "Compendium.pf2e.conditionitems.Unconscious"
```

Comment:
We should not grant conditions.

Reply:
Fair. Is this because conditions are to be refactored? I added these because some cards give conditions that last x rounds and it can be hard to remember (sometimes several fumbles/crit effects happen in a single round of combat)

Can this be an effect with no rules and just the label/effect duration? (Although I heard we don't want effect timers without rules?)

Or is there any other suggestions on how to "track" conditions?